heroku/cli
# Jw/redis v5/migrate redis cli

This PR migrates `redis:cli` to TypeScript and relocates it in the `pacakges/cli` directory.

## Test
1. Checkout this branch and run `yarn && yarn build`
2. Add a redis instance to your app if none exists: `heroku addons:create -a YOUR_APP_ID heroku-redis:premium-0`
3. Wait for provisioning to complete.
4. Connect to the redis instance: `./bin/run redis:cli -a YOUR_APP_ID` 
5. If more than 1 redis-heroku database exists, specify the database to connect to `./bin/run redis:cli -a YOUR_APP_ID DATABASE`
6. Observe the console is now showing a connection to the redis instance.
